src/nl/hannahsten/texifyidea/util/Magic.kt

```diff
@@ -410,6 +410,14 @@ object Magic {
             "\\pnotecite", "\\fnotecite"
         )
 
+        /**
+         * All commands that represent a reference to a bib entry, including user defined commands.
+         */
+        fun getBibRefereneCommands(project: Project): Set<String> {
```

Comment:
A util file might be better suited for such a(n extension?) function. same goes for getLabelDefinitionCommands. Magic is mostly for constants.

Reply:
I agree... I suppose I just followed `getLabelReferenceCommands`. I'll move all of them to a util file.